### ISSUE TITLE
Meta-request: please use the Github 'Releases' functionality for better notifiabiliy

### ISSUE THREAD

**Issue by a10kiloham** [bigtreetech/PandaTouch#150]
Sites like https://newreleases.io/ track when there is a new release added and can notify. It's hard to tell when you add a new release here otherwise, it requires setting a crawler up on your wiki.
Thank you!

**Comment by BTTJUSTIN** [comment]
Thanks for making this suggestion. We will discuss it with our engineer.

**Comment by UsedOak** [comment]
Yesssss

**Comment by dilixi** [comment]
> Sites like https://newreleases.io/ track when there is a new release added and can notify. It's hard to tell when you add a new release here otherwise, it requires setting a crawler up on your wiki. Thank you!

Hi,
Did you have time to do a test for this?
we can do a test with this rep:
https://github.com/bigtreetech-beta/PandaTouch-Support

And please subscribe to events for bigtreetech-beta/PandaTouch-Support

![40608365-1E32-42c2-A743-224065759196](https://github.com/user-attachments/assets/e3fe5a34-1057-4fb5-b2e2-0001a91e268b)

You can email me when you have time.

1284513393@qq.com

**Comment by a10kiloham** [comment]
Yes I can do that notification but I only want notification when there is a release not idle chatter. You still haven't set up releases yet properly.

**Comment by dilixi** [comment]
> Yes I can do that notification but I only want notification when there is a release not idle chatter. You still haven't set up releases yet properly.

I mean that you need to do something before you can get this notification.
you need to follow these steps:
1 open this link
https://github.com/bigtreetech-beta/PandaTouch-Support

2 click the 'watch' button and click the 'custom' button
![image](https://github.com/user-attachments/assets/2a125a51-627c-4aeb-ba49-9399965b3471)

3 choose the 'releases'
![image](https://github.com/user-attachments/assets/89e169ae-acd7-4b1b-90aa-2192ab0da8e7)

4 give me a email and I will add a release on PandaTouch-Support

5 check your email

I think it is very easy for you to do that test.

**Comment by dilixi** [comment]
> Yes I can do that notification but I only want notification when there is a release not idle chatter. You still haven't set up releases yet properly.

@a10kiloham 

I mean that you need to do something before you can get this notification.
you need to follow these steps:
1 open this link
https://github.com/bigtreetech-beta/PandaTouch-Support

2 click the 'watch' button and click the 'custom' button
![image](https://github.com/user-attachments/assets/2a125a51-627c-4aeb-ba49-9399965b3471)

3 choose the 'releases'
![image](https://github.com/user-attachments/assets/89e169ae-acd7-4b1b-90aa-2192ab0da8e7)

4 give me a email and I will add a release on PandaTouch-Support

5 check your email

I think it is very easy for you to do that test.

**Comment by maple-underscore** [comment]
Any update on this? I also think that releases would be a better system than having to go through the firmware folder to see if there is a new one.

**Comment by bigtreetech** [comment]
Should be coming with 1.0.7 @dilixi
